typescript/vrotsc/e2e/cases/saga/testConditionalExecution.saga.yaml

```diff
@@ -0,0 +1,15 @@
+id: 3669e-e7ea-4486-8b91-e1e36ae7ce9
+name : Test Conditional Execution Saga
+path: sagas/saga-test-conditional-execution
+imports:
+  - saga.test.saga.tasks
+attributes:
+  conditionalVar:
+    type: boolean
+
+tasks:
```

Comment:
Would be good to add test for workflows as well, ideally separately, so we can figure out where the error is if it happens faster